runtime/decimal.c

```diff
@@ -107,6 +109,37 @@ TaggedPtrPanicCode finish(decQuad *dq, decContext *cx) {
     return result;
 }
 
+bool _bal_decimal_exact_eq(TaggedPtr tp1, TaggedPtr tp2) {
+    decQuad dc1;
+    decQuad dc2;
+    decQuadCanonical(&dc1, taggedToDecQuad(tp1));
+    decQuadCanonical(&dc2, taggedToDecQuad(tp2));
+    return memcmp(&dc1, &dc2, DECQUAD_Bytes) == 0;
+}
+
+int64_t _bal_decimal_cmp(TaggedPtr tp1, TaggedPtr tp2) {
+    decQuad d;
+    decContext cx;
+    initContext(&cx);  
+    decQuadCompare(&d, taggedToDecQuad(tp1), taggedToDecQuad(tp2), &cx);
+    // The return value of decQuadCompare can be 1, 0, -1 or NaN.
```

Comment:
Why are we using decClass rather than simply returning the result of decQuadCompare?

Reply:
@jclark, I am returning `int64_t` to be consistent with other runtime cmp functions.
Since `decQuadCompare` returning decQuad as its' result, I used decClass.

Is that okay to return result of `decQuadCompare` which is a decQuad?